inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/ProxySink.java

```diff
@@ -104,11 +103,10 @@ public void write(Message message) {
                                 }
                                 // add message to package proxy
                                 packProxyMessage.addProxyMessage(proxyMessage);
-                                //
+                                // update msgTime
+                                packProxyMessage.updateMsgTime(System.currentTimeMillis());
```

Comment:
It is better to use current time in Agent source.

Reply:
Miss the modification about AUDIT_ID_AGENT_READ_SUCCESS.

Reply:
> It is better to use current time in Agent source.

In `agent-source`, messages are encapsulated piece by piece. But, they are packaged to be sent to `dataproxy` with only one  `msgTime`. So, is it better to use current time here?